redux/longestStreakSlice.ts

```diff
@@ -0,0 +1,71 @@
+import { createSlice, PayloadAction, Slice } from "@reduxjs/toolkit";
+import { RootState } from "./rootReducer";
+import { BlockState } from "../components/math/longestStreak/MultiplicationBlock";
+import { GameBlockState } from "../pages/studentPortal/labs/multiplication/game";
+import { STAGE } from "../pages/studentPortal/labs/multiplication/game";
+
+export interface LongestStreakState {
+  stage: number;
+  blocks: GameBlockState[];
+  currentBlock: number;
+  isPlayerOneActive: boolean;
+  showPlayerOneName: string;
+  showPlayerTwoName: string;
+  isFilledBoard: boolean;
```

Comment:
if you're not using this field yet, I would remove it until you actually need it